runok.js

```diff
@@ -218,6 +218,7 @@ Our community prepared some valuable recipes for setting up CI systems with Code
         if (!text) return;
         cfg.replace('<!-- configuration -->', text[1]);
         cfg.replace(/## config((.|\n)*)\[1\]/m, '');
```

Comment:
This regex was the issue @DavertMik it is somehow replacing from ## config to [1]. That's why there is only : left and causes the issue.